tutorials/weaviate_llama_index_search_and_retrieval.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    url = "http://storage.googleapis.com/arize-assets/xander/milvus-workshop/milvus_dataset.json"
</pre><p>Change you change this link to http://storage.googleapis.com/arize-assets/xander/arize_docs_corpus.json?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>